airflow/providers/weaviate/hooks/weaviate.py

```diff
@@ -94,18 +93,14 @@ def get_conn(self) -> WeaviateClient:
             url=url, auth_client_secret=auth_client_secret, additional_headers=additional_headers
         )
 
+    @cached_property
     def get_client(self) -> WeaviateClient:
```

Comment:
I'm not sure whether the name `client` makes more sense here, but if we're to go with `client`. It means we still need to deprecate this. and `conn` might be an even better name in that case

Reply:
We already have a `get_conn`. This is just to serve as proxy for it. I think some other providers do this. I will check and revert

Reply:
yep, what I meant is something like 

```python
    @cached_property
    def conn(...):
```

Reply:
https://github.com/apache/airflow/blob/main/airflow/providers/amazon/aws/hooks/redshift_sql.py#L73-L75